README.md

```diff
@@ -70,8 +68,10 @@ _Try this code interactively using [IPython](https://ipython.org/install.html)._
 
 ## Installation
 
+_This project is still maturing. Be sure to pin any dependencies to the latest minor._
```

Comment:
might want to update the trove classifiers to reflect this

Reply:
We are already pointing at `Status :: Alpha`. Anything else we should do?

Reply:
`"Development Status :: 4 - Beta"`

Reply:
True. :)